apps/sms/test/marionette/conversation_test.js

```diff
@@ -491,4 +503,83 @@ marionette('Conversation Panel Tests', function() {
       });
     });
   });
+
+  suite('Add recipients from thread', function() {
+    var conversationView;
+    setup(function() {
+      var thread = ThreadGenerator.generate({
+        numberOfMessages: 1
+      });
+      messagesApp.launch();
+
+      storage.setMessagesStorage([thread], ThreadGenerator.uniqueMessageId);
+      // Set empty contacts store.
+      storage.setContactsStorage();
+      var inboxView = new InboxView(client);
+      conversationView = inboxView.goToConversation(thread.id);
+      messagesApp.Composer.messageInput.sendKeys('message2');
+    });
```

Comment:
yes, you need node 4 and clang now :)

please ping me on IRC if you still have issues !